css-three-column-layout/styles.css

```diff
@@ -0,0 +1,67 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 900px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+body {
+  background: #109ee5;
+  font-family: Lato, sans-serif;
+  margin: 0;
+}
+
+h2 {
+  color: white;
+  margin-top: 5em;
+  text-align: center;
+  font-size: 30px;
+}
+
+h3 {
+  color: white;
+  font-size: 20px;
+}
+
+section {
+  text-align: center;
+  width: 90%;
+}
+
+p {
+  color: lightgray;
+  font-size: 15px;
+  line-height: 1.5em;
+}
+
+img {
+  width: 60%;
+  margin: auto;
+}
+
+.circle {
+  background: white;
+  border-radius: 50%;
+  height: 100px;
+  width: 100px;
+  justify-content: center;
+  padding-top: 25px;
+  padding-left: 25px;
+  margin-left: 75px;
+  margin-top: 1em;
```

Comment:
instead of using all this margin and padding to position the circle, I would use `display flex` on the column parent. 

You also might want to consider putting `display-flex` on this class so that you can center the image in the circle using flex.